### ISSUE TITLE
middleman detector: hasRequiredFiles is not defined

### ISSUE THREAD

**Issue by ridem** [netlify/netlify-dev-plugin#177]
This detector doesn't require the `hasRequiredFiles` util, so calls to `netlify dev` fail.
[src/detectors/middleman.js#L4](https://github.com/netlify/netlify-dev-plugin/blob/47ed3e6fe644a4463a843bb9f2c3494eb0fb82ec/src/detectors/middleman.js#L4)

Bug introduced in PR #172 

```
◈ Netlify Dev ◈
ReferenceError: hasRequiredFiles is not defined
    at Array.module.exports (.../netlify-cli/node_modules/netlify-dev-plugin/src/detectors/middleman.js:4:3)
    at module.exports.serverSettings (.../netlify-cli/node_modules/netlify-dev-plugin/src/detect-server.js:20:40)
    at DevCommand.run (.../netlify-cli/node_modules/netlify-dev-plugin/src/commands/dev/index.js:187:26)
```

netlify-cli/2.11.17 darwin-x64 node-v10.13.0 - MacOS 10.14.4

**Comment by lotas** [comment]
Stumbled upon the same problem just now.
Guess it was caused by this recent change https://github.com/netlify/netlify-dev-plugin/commit/340a44291c7bc21b3c52609434580c44d77e1c2d
It expects `config.rb` somewhere

**Comment by sw-yx** [comment]
sorry for the bug. fixing

**Comment by sw-yx** [comment]
fixed release in v2.11.18, thanks @jgarber623 !
